app/helpers/application_helper.rb

```diff
@@ -8,11 +8,25 @@ module ApplicationHelper
     {:name => "Contact", :link => "contact"}
   ]
 
+  # For pages not included in the main menu that need their headings displayed in the top header
+  PAGES = [
+    {:name => "Documents", :controller => "documents"}
+  ]
+
   def get_active_menu_item
     item = MENU.find{|m| m[:link].gsub('-', '_') == params[:controller]}
     item.nil? ? "" : item[:name]
   end
 
+  def get_active_page
```

Comment:
These two methods repeat each other. Feel free to revise this in a future update.